templates/index.html

```diff
@@ -4,7 +4,6 @@
 {% endblock %}
 
 {% block takeover_content %}
-  {% include "takeovers/_multicloud-operations.html" with nojs_fallback=True %}
```

Comment:
According to the takeover schedule, this should look like:   
```
  {% include "takeovers/_robotics-takeover.html" with nojs_fallback=True %}
  {% include "takeovers/_multicloud-operations.html" %}
  {% include "takeovers/_financial-services.html" %}
```

Reply:
Thanks, I set that to match the content in the issue: https://github.com/canonical-websites/www.ubuntu.com/issues/3983

Reply:
@shivjoshi1996 do you know we changed the alternating takesovers order for this one?

Reply:
@caldav @anthonydillon On the takeover schedule it says that this takeover is supposed to alternate with the IoT App Store one, but I now see that Marketing have added a new section on the takeover spreadsheet that has the order David has described. I'll sync up with Marketing tomorrow and finalise an order and make sure it's conveyed a bit more clearly on their spreadsheet. Will let you know once I've talked to everyone.

Reply:
The order is not that important IMHO. The reasoning is to have - when possible - an IoT/devices takeover, a cloud one and a more global campaign, for the sake of diversity of messages. Also, the `nojs_fallback` one is really for a small portion of our traffic, and the more technical takeover is preferred (with the assumption that having JS disabled is mostly a technically focused visitor thing).

Reply:
@caldav Yeah I agree, the reasoning definitely makes sense. When I wrote the initial issue the only alternating takeover was IoT, as the other section wasn't there yet, so that's where the confusion was.